niri-ipc/src/lib.rs

```diff
@@ -457,6 +467,16 @@ pub enum Action {
     MoveWindowToMonitorPrevious {},
     /// Move the focused window to the next monitor.
     MoveWindowToMonitorNext {},
+    /// Move the focused window to a specific monitor.
+    #[cfg_attr(
+        feature = "clap",
+        clap(about = "Move the focused window to a specific monitor")
+    )]
+    MoveWindowToMonitor {
+        /// The target output name.
```

Comment:
Do you mind also adding an `--id` argument here like in other Window actions? (at least most ones)

Reply:
Looks doable! :)